modules/apps/object/object-test/src/testFunctional/macros/ObjectAdmin.macro

```diff
@@ -1028,6 +1026,8 @@ definition {
 						value1 = ${inputExpressionValue});
 				}
 
+				var key_text = "Message";
```

Comment:
You don't need to declare the variable here, just pass the attribute in the conditional: `if (IsElementPresent(locator1 = "TextInput#ANY", key_text = "Message")) {`